### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by BrendenSeifried**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,15 @@
...
-
--   Ask: which of your HTML elements need to be hard coded, and which need to be dynamically generated?
--   Consider your data model.
-    -   What kinds of objects (i.e., Dogs, Friends, Todos, etc) will you need?
-    -   What are the key/value pairs?
-    -   What arrays might you need?
-    -   What needs to live in a persistence layer?
--   Is there some state we need to initialize?
--   Ask: should any of this work be abstracted into functions? (i.e., is the work complicated? can it be reused?)
+Things changed and fell out of order but I got it done none the less!
```
Comment (juliannevela) [inline README.md:15]: That is the nature of development, my friend! Things will never go exactly as you have planned them, but the important thing is that you had a basic outline plan established that gave you a path to complete your goal.

File: index.html [hunk 2]
```diff
@@ -12,17 +12,32 @@
...
+                My favorite kinds of animals!
+                <ul id="animals">
+                    <li>Dog!</li>
+                    <li>Cappybara</li>
+                    <li>seals</li>
+                </ul>
+            </section>
+            <section>
+            <p>Fun Fact: I have seen many many UFO's. Starting from when I lived in Nevada to many in Washington!</p>
+            <p>I have seen lights move at high speed and change direction in the sky to full low flying craft in broad daylight.</p>
```
Comment (juliannevela) [inline index.html:36]: Uhhmmm......this is AWESOME! I've only ever seen maybe 1-2 UFO's that I can specifically remember.

**Review by juliannevela — COMMENTED** [review]
Great work! You nailed this one! :)

In the future, please leave the PR open until we've graded it, though. You'll have the deploy preview to access from within your PR so you can test your changes live if needed.

**Comment by BrendenSeifried** [comment]
Thank you for the words of encouragement! I'm trying to follow as best I
can and I know it's a process that will eventually sink in with repetition
but knowing I did well with this is a huge boost. I still cannot believe
that the government said "yeah UFO's are real and area 51 is in fact a
military base" and most everyone didn't even bat an eyelash. I feel like we
live in bizarro world now, but if you ever get time to talk about UFO's and
your experience with them I'm all for it.

Thanks again.
Brenden

**Pull request merged**